packages/js/product-editor/jest.config.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Nice!  Just a heads up that we'll probably have conflicts (or hopefully just need a rebase) if [this](https://github.com/woocommerce/woocommerce/pull/37225) is merged first, but we'll probably merge your PR first anyway.

Reply:
Thanks, it as totally expected!